### PR TITLE
Easy iOS hockey-app distribution

### PR DESCRIPTION
**Pull request by DanielLeushuis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,11 +55,29 @@ npm run release:android
...
+```
+
+For this to work the environment variable `HOCKEY_APP_TOKEN` is expected to be available.
+
+### Building & uploading a test version to Hockey App for iOS
+
+To build and upload at the same time
+
+```
+npm run release-upload-hockey-app:android
```
Comment (Mathyn) [inline README.md:78]: Shouldn't this be `npm run release-upload-hockey-app:ios`?

File: src/core/merkle/MerkleTreeBuilder.ts [hunk 2]
```diff
@@ -64,8 +64,12 @@ export class MerkleTreeBuilder {
...
                 ];
-            }
-            else {
+            } else if (isIos) {
+                scripts = [
+                    `${ window.location.href.replace("/index.html", "") }/assets/scripts/sjcl.js`,
+                    `${ window.location.href.replace("/index.html", "") }/assets/scripts/seedrandom.min.js`
+                ];
+            } else {
                 // Web & ios is easy.
```
Comment (Mathyn) [inline src/core/merkle/MerkleTreeBuilder.ts]: This comment is no longer correct

**Review by Mathyn — CHANGES_REQUESTED** [review]
Looking very good but please take a look at these minor issues I found.

File: README.md (hunk 1 above)
Comment (DanielLeushuis) [reply README.md:78]: Good catch! The command was right but the title was wrong

**Review by Mathyn — APPROVED** [review]

**Pull request merged**